packages/character-count-ui/src/CharacterCounter.tsx

```diff
@@ -25,7 +25,7 @@ const calculateCharCount = (editor?: TEditor) => {
 }
 
 export function getCharacterCount(id: string): number {
-  const editor = useStoreEditorState(id)
+  const editor = useStoreEditorRef(id)
```

Comment:
this is a hotfix for selection, now when you double click a word and try to go backwards(to left) , it's gonna work properly